frontend/packages/ceph-storage-plugin/src/components/create-backingstore-page/backing-store-dropdown.tsx

```diff
@@ -23,19 +24,23 @@ export const BackingStoreDropdown: React.FC<BackingStoreDropdownProps> = ({
 }) => {
   const { t } = useTranslation();
   const [isOpen, setOpen] = React.useState(false);
-  const [nbsObj, nbsLoaded, nbsErr] = useK8sGet<ListKind<BackingStoreKind>>(
-    NooBaaBackingStoreModel,
-    null,
+
+  const nooBaaBsResource = {
```

Comment:
Move it to `resource.ts`
```suggestion
  const backingStoreResource = {
```